README_CN.md

```
@@ -20,18 +20,472 @@ Getty 是一个使用 Golang 开发的异步网络 I/O 库。它适用于 TCP、
+
+**属性管理**
+- **`GetAttribute(key any)`**: 获取会话属性
+- **`SetAttribute(key any, value any)`**: 设置会话属性
+- **`RemoveAttribute(key any)`**: 删除会话属性
+
+**统计信息**
+- **`Stat()`**: 获取会话统计信息（连接状态、读写字节数、包数量等）
+
+#### 活跃时间更新机制
+
+**自动活跃时间更新**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)